server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -305,6 +314,38 @@ def get_sql(self, **kwargs: Any) -> str:
         # without actually executing the query
         return custom_query, StepTable(columns=["*"])
 
+    # def cumsum(
```

Comment:
If this cannot be implemented as is, should this comment really live here ? I'd rather have it in the snowflake translator.

Reply:
great thx